src/js-feature-toggle-manager.ts

```diff
@@ -7,7 +7,7 @@ export interface ToggleStatus {
 
 export class FeatureToggleManager {  
   private static _instance: FeatureToggleManager;
-  private currentToggles: ToggleStatus[];
+  private _toggles: ToggleStatus[];
```

Comment:
Should not be `_toggles` as this is not the coding standard. What is the reason for needing to change it from currentToggles to toggles?